production/helm/loki/templates/backend/query-scheduler-discovery.yaml

```diff
@@ -9,6 +9,13 @@ metadata:
   labels:
     {{- include "loki.backendSelectorLabels" . | nindent 4 }}
     prometheus.io/service-monitor: "false"
+  annotations:
+    {{- with .Values.loki.serviceAnnotations }}
+    {{- toYaml . | nindent 4}}
+    {{- end }}
+    {{- with .Values.backend.service.annotations }}
```

Comment:
The `query-scheduler-discovery` service is not _technically_ the backend service, but given it's deployed along with it and currently uses the `backendSelectorLabels`, this seems appropriate to add these annotations as well.